Makefile

```diff
@@ -99,6 +99,7 @@ helm: ## Generate Helm Chart
 
 website: ## Generate Docs Website
 	cp -r website/content/en/preview website/content/en/${RELEASE_VERSION}
+	find website/content/en/${RELEASE_VERSION}/ -type f | xargs  perl -i -p -e "s/{{< param \"latest_release_version\" >}}/${RELEASE_VERSION}/g;"
```

Comment:
Can you use yq for this?

Reply:
no, this is the actual markdown file not a config file. Unless there's something I don't know about `yq` 👀